### PR TITLE
Updated Healthcare Service profile

### PR DESCRIPTION
**Pull request by arunsundar-m**
1. Added slice under type element
2. Bound signal-healthcare-service category to HealthcareService.category

### CODE REVIEW

**Comment by arunsundar-m** [comment]
@enjoysparkling Created the PR, please review.

File: input/resources/StructureDefinition-profile-SignalHealthcareService.json [hunk 1]
```diff
@@ -12,6 +12,14 @@
   "derivation": "constraint",
   "differential": {
     "element": [
+      {
+        "id": "HealthcareService.category",
+        "path": "HealthcareService.category",
+        "binding": {
+          "strength": "example",
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-profile-SignalHealthcareService.json:19]: Change strength to 'extensible' at least, but it is 'required' for our model in all situations, isn't it?

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
see in-line notes:
* Change binding on category to required.

File: input/resources/StructureDefinition-profile-SignalHealthcareService.json (hunk 1 above)
Comment (Gokulsignal) [reply input/resources/StructureDefinition-profile-SignalHealthcareService.json]: Updated the Binding Strength as required for "StructureDefinition-profile-SignalHealthcareService". 

https://github.com/enjoysparkling/signal-mso-fhir-profiles/commit/b6d358cc8c9ab6085bbed60573d4f7689d053a65

**Review by enjoysparkling — APPROVED** [review]

**Pull request merged**